features/admin.applications.v1/components/forms/inbound-oidc-form.tsx

```diff
@@ -2603,6 +2695,125 @@ export const InboundOIDCForm: FunctionComponent<InboundOIDCFormPropsInterface> =
                                 readOnly={ readOnly }
                                 data-testid={ `${ testId }-access-token-type-radio-group` }
                             />
+                            { isJWTAccessTokenTypeSelected ? (
+                                <Grid.Row>
+                                    <Grid.Column width={ 8 }>
+                                        <Autocomplete
+                                            disablePortal
+                                            multiple
+                                            disableCloseOnSelect
+                                            loading={ isLoading }
+                                            options={ accessTokenAttributes }
+                                            value={ selectedAccessTokenAttributes ?? [] }
+                                            disabled = { !initialValues?.accessToken?.accessTokenAttributesEnabled }
+                                            data-componentid={ `${ componentId }-assigned-jwt-attribute-list` }
+                                            getOptionLabel={
+                                                (claim: ExternalClaim) => claim.claimURI
+                                            }
+                                            renderInput={ (params: AutocompleteRenderInputParams) => (
+                                                <>
+                                                    <InputLabel htmlFor="tags-filled" disableAnimation shrink={ false }>
+                                                        { t(
+                                                            "applications:forms.inboundOIDC.sections" +
+                                                            ".accessToken.fields.accessTokenAttributes.label"
+                                                        ) }
+                                                    </InputLabel>
+                                                    <TextField
+                                                        className="jwt-attributes-dropdown-input"
+                                                        { ...params }
+                                                        placeholder={ !false &&
+                                                            t("applications:forms.inboundOIDC.sections" +
+                                                        ".accessToken.fields.accessTokenAttributes.placeholder") }
```

Comment:
This will always be evaluated to the placeholder text.

```suggestion
                                                        placeholder={ t("applications:forms.inboundOIDC.sections" +
                                                        ".accessToken.fields.accessTokenAttributes.placeholder") }
```

Reply:
fixed